src/flb_config_map.c

```diff
@@ -649,10 +649,10 @@ int flb_config_map_set(struct mk_list *properties, struct mk_list *map, void *co
         }
         else if (m->type == FLB_CONFIG_MAP_TIME) {
             m_i_num = (int *) (base + m->offset);
-            *m_i_num = m->value.val.s_num;
+            *m_i_num = m->value.val.i_num;
```

Comment:
The value is stored in the first 4 byte of `val`, get it using s_num will add 4 bytes of zero on the higher address in memory, this gives different result on big endian systems and little endian systems, this can be proved using the below code
```
#include <stdio.h>
#include <stddef.h> // Include this header for size_t

int main() {
    printf("Size of size_t: %zu bytes\n", sizeof(size_t));
    printf("Size of int:    %zu bytes\n", sizeof(int));

    int a[2] = {1, 0};
    size_t *s_num = (size_t *)a;
    printf("%zu\n", *s_num);

    int *i_num = (int *)a;
    printf("%d\n", *i_num);
    return 0;
}
```
The output of the code on x86(little endian system):
```
Size of size_t: 8 bytes
Size of int:    4 bytes
1
1
```
On s390s system(big endian):
```
Size of size_t: 8 bytes
Size of int:    4 bytes
Value of s_num: 4294967296
Value of i_num: 1
```

Reply:
The s_num on big endian system 4294967296 equals to `1<<32`

Reply:
Please revert this.